### PR TITLE
Fix samples of DynamoDBEvent and SNSEvent

### PR DESCRIPTION
**Pull request by budougumi0617**
I found incorrect upper/lower case and syntax error in sample codes (My changes have results of gofmt).
I would appreciate it if you could check this changes.

### CODE REVIEW

File: events/README_DynamoDB.md [hunk 1]
```diff
@@ -3,27 +3,25 @@
...
-func handleRequest(ctx context.Context, e events.DynamoDbEvent) {
+func handleRequest(ctx context.Context, e events.DynamoDBEvent) {
 
-    for _, record := range e.Records {
-        fmt.Printf("Processing request data for event ID %s, type %s.\n", record.EventId, record.EventName)
+	for _, record := range e.Records {
+		fmt.Printf("Processing request data for event ID %s, type %s.\n", record.EventID, record.EventName)
 
-        // Print new values for attributes of type String
-        for name, value := range record.Dynamodb.NewImage {
```
Comment (budougumi0617) [inline events/README_DynamoDB.md]: I think correct name is `Change`.

https://github.com/aws/aws-lambda-go/blob/6e2e37798efbb1dfd8e9c6681702e683a6046517/events/dynamodb.go#L18

File: events/README_SNS.md [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
 
 ```go
-
 import (
-    "strings"
-    "github.com/aws/aws-lambda-go/events")
+	"context"
+	"fmt"
 
-func handler(ctx context.Context, events.SnsEvent snsEvent) {
```
Comment (budougumi0617) [inline events/README_SNS.md]: Correct syntax is `snsEvent events.SNSEvent`

File: events/README_DynamoDB.md [hunk 3]
```diff
@@ -3,27 +3,25 @@
...
-        	if value.DataType() == ddbevents.DataTypeString {
```
Comment (budougumi0617) [inline events/README_DynamoDB.md]: `DataTypeString` is defined in `events` package.
https://github.com/aws/aws-lambda-go/blob/28a6e9fa1fc02d45d9a8c0a8d22c7d5c3580aa03/events/attributevalue.go#L142

**Comment by bmoffatt** [comment]
Thank you for the doc fixes!

**Pull request merged**